### ISSUE TITLE
shift signup: only list user of the selected angeltype

### ISSUE THREAD

**Issue by rootiam** [engelsystem/engelsystem#575]
When you want to sign up an angel for a shift after clicking "sign up" for a shift in the main shift view, you can choose the angeltype and the user to assign.
I think it would help if only the users of the selected angeltype would appear.
Currently I can assign a user with an angeltype that the user never selected to be available for.

**Comment by MyIgel** [comment]
Afaik this is intended for shift coordinators (ping @msquare?), but it would be nice to have a select where they are at least separated / grouped / searchable

**Comment by msquare** [comment]
Yep, this is intended. Some helper is often better than no helper, so in chaotic situations a shift coordinator must be able to pick any angel and assign a shift as any type.

Maybe the unholy dropdown containing all the users should list the users in the matching angeltype first and then all the others.
